.gitignore

```diff
@@ -13,4 +13,6 @@ tmp/
 # Default Waypoint database file
 data.db
 
-.vscode
+# .vscode - We intentionally commit .vscode in order to share project-specific settings
```

Comment:
But what about all of my personal vscode settings? Have you shared any how-tos on how to keep them separate? I can look it up of course but it'd be nice if that advice was part of this :)

Reply:
Although, maybe this would be an OK thing to have. It would encourage us to centralize some of the tooling with our project-specific settings. My project vscode settings right now just have a `launch.json` file that @jgwhite shared with me, and that might be OK to get committed. What do you think Jamie?

Reply:
Agreed. @krantzinator I’d love to learn some tips and tricks from your local `.vscode` 🙂

Reply:
And yeah, I think sharing that `launch.json` would be very useful indeed. It’s akin to sharing the `hack` directory.